### DISCUSSION TITLE
Need help with LabWC in Fedora

### DISCUSSION THREAD

**Discussion by glow12121** [labwc/labwc#1187] [Q&A]
Hi,

I want to install LabWC alongside the default Gnome in Fedora 38/39. What modules should I be looking for to get a usable system OS?

Something like regular old-school Windows, or Budgie?

**Comment by stefonarch** [comment]
Depends on your needs, panels, menus ecc.
 I think you saw the docs like https://labwc.github.io/integration.html 

Personally I've set up an (mostly) Qt environment using most of LXQt: https://github.com/stefonarch/lxqt-labwc-session but when you come from gnome you may prefer to stay with GTK.

**Comment by Consolatis** [comment]
Most important, make sure to also install xwayland on fedora, there was a bug with their packaging that built labwc with xwayland support but didn't set it as dependency for the labwc package. I don't know if that has been fixed.

**Comment by glow12121** [comment]
@Consolatis @stefonarch Thanks, I will try this out!
